tests/hpc/mpi_master.pm

```diff
@@ -122,7 +129,10 @@ sub run ($self) {
             assert_script_run($mpirun_s->slave_nodes("$exports_path{'bin'}/$mpi_bin"), timeout => 120);
             assert_script_run($mpirun_s->n_nodes("$exports_path{'bin'}/$mpi_bin", 2), timeout => 120);
         } else {
-            assert_script_run($mpirun_s->all_nodes("$exports_path{'bin'}/$mpi_bin"), timeout => 120);
+            # Skipping papi test on compute nodes for now
```

Comment:
Can we just add that that this is likely due to the source file?

Reply:
Issue is with the loading of `papi` module itself  with current flow of execution in compute nodes, I don't think it is due to the source file.